### DISCUSSION TITLE
Brace Match alpha

### DISCUSSION THREAD

**Discussion by gabdub** [orbitalquark/textadept#175] [Q&A] [answered]
Hi,
I noticed that the alpha of matching brackets indicator changed in 11.2 (the color is OK but it's dimmer), and it seems that

`view.indic_alpha[textadept.editing.INDIC_BRACEMATCH] = 255`

doesn't change anything... any ideas?

![brackets](https://user-images.githubusercontent.com/11639609/155203326-2dd2d206-d997-422a-8859-1d2357fce5b2.png)

**Comment by orbitalquark** [comment]
Do you mind trying to pinpoint the beta release that this changed in?

**Comment by gabdub** [comment]
Sure. Here it is:

![beta](https://user-images.githubusercontent.com/11639609/155229090-8f646ed4-5014-4a95-a8a4-f0669128365c.png)

![beta2](https://user-images.githubusercontent.com/11639609/155229114-1570a33c-b0a0-4b44-bd78-78f5374e42c5.png)

**Comment by gabdub** [comment, marked as answer]
It seems that the default changed but it can be restored using:
`view.indic_outline_alpha[textadept.editing.INDIC_BRACEMATCH] = 255`

**Comment by orbitalquark** [comment]
Thanks for your help. I will commit a change for this in the theme files. It will be in the next release.
